.golangci.yml

```diff
@@ -26,7 +26,7 @@ linters:
     # The rules violated to make lint work are commented out. We will fix it eventually.
     - dupl
     - errcheck
-    - exportloopref
+    - copyloopvar
```

Comment:
I couldn't find the reason for this change. Could you tell me why this change was caused?

Reply:
Please see the commit message. https://github.com/topolvm/topolvm/pull/978/commits/368fd9c7d73dde01cc26a84bf1f2686e36bce04c
golintci-lint ouputs the deprecation message.